hledger/Hledger/Cli/Commands/Print.md

```diff
@@ -29,6 +29,7 @@ Flags:
                             txt, beancount, csv, tsv, html, fods, json, sql.
   -o --output-file=FILE     write output to FILE. A file extension matching
                             one of the above formats selects that format.
+  -l --location             add file/line number tags to print output.
```

Comment:
This will get regenerated by ./Shake cmddocs later, but no harm to add it now.

Let's also remove bin/hledger-print-location.hs, and move its doc from bin/README.md to Print.md > "print, other features".